docs/content.md

```diff
@@ -442,3 +445,64 @@ When updating not just the text content within the file, but the actual list of
 - Export the XLSX to CSV with filename `internship_providers.csv`
 - Place it in the `lib/tasks/support` directory.
 - Run `bundle exec rake teaching_internship_providers:generate`.
+
+
+## Creating a new page
+
+1. **Make sure you start by creating a new branch on master**
+2. Navigate to the right hand side bar, where you want the new page to sit
+3. Right hand click and select 'New file'
+4. Give name to file ie. when-to-apply.md **make sure you put md on the end of the name if it is a markdown file**
+
+Now you have created a page - you will need to fill in the top details:
+
```

Comment:
How do I add in the different components? Do I need to copy them from existing pages? How do I know what components I need and how do I order them? How do I add in things like CTAs?

Reply:
@MaxineCl probably not worth putting in here because i've never done it/always ask gareth to do it?

Reply:
I think this is worth adding - this is useful to know as you should be able to create your own pages?

Reply:
I'd suggest you create a new ticket that we can come back to in early January and list the other things to add, rather than delay this PR